Add more handlers

mxmehl opened the pull request:
This PR adds a few more handlers which may be useful in some scenarios.

mxmehl committed:
add more handlers

gekmihesg commented:
Sure, but using the `nohup` module on those isn't really needed. I did that on the others, because they are very likely to break the network connection to the Ansible controller which would cause errors. Those you've added look safe to directly use `uci` or `service` instead.

gekmihesg commented on handlers/main.yml:
```diff
@@ -28,3 +28,21 @@
     - restart network
+
+- name: uci commit
+  nohup:
```
Please use `uci` directly here

gekmihesg commented on handlers/main.yml:
```diff
@@ -28,3 +28,21 @@
+  when: "'openwrt' in group_names"
+
+- name: restart firewall
+  nohup:
```
Please use `service`

gekmihesg commented on handlers/main.yml:
```diff
@@ -28,3 +28,21 @@
+  when: "'openwrt' in group_names"
+
+- name: reload system
+  nohup:
```
Please use `service`

mxmehl committed:
force using legacy scp protocol instead of scp

zeerayne committed:
Add compatibility with ansible >= 2.19

In ansible 2.19 release return value of `ActionBse._configure_module`
function was changed which leads to raising `Task failed: not enough
values to unpack (expected 4, got 2)` error on any role task run.

This change add compatibility with ansible >= 2.19 and preserves
compatibility with older versions.

zeerayne committed:
Conditionals must have a boolean result

mxmehl committed:
Merge upstream PR #77 before upstream merge

Add compatibility with ansible >= 2.19

URL: https://github.com/gekmihesg/ansible-openwrt/pull/77
